config.xml

```diff
@@ -23,7 +23,7 @@
     <access origin="*" />
     <allow-intent href="http://*/*" />
     <allow-intent href="https://*/*" />
-    <preference name="android-minSdkVersion" value="22" />
+    <preference name="android-minSdkVersion" value="23" />
```

Comment:
Why do we need to upgrade, and what are the implications in user numbers?

Reply:
Too many things to track. I found the details, the deadline is August 31, 2023, so we have to do it anyway.